x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.ts

```diff
@@ -102,7 +102,7 @@ export const createLogThresholdExecutor = (libs: InfraBackendLibs) =>
     LogThresholdAlertState,
     LogThresholdAlertContext,
     LogThresholdActionGroups
-  >(async ({ services, params, spaceId, startedAt }) => {
+  >(async ({ services, params: ruleParams, spaceId, startedAt }) => {
```

Comment:
nit: is that renaming useful?

Reply:
A leftover of the first attempt I did, I'll switch it back!